daml-assistant/integration-tests/src/Main.hs

```diff
@@ -189,10 +206,21 @@ quickstartTests quickstartDir mvnDir = testGroup "quickstart"
     where
         mvnRepoFlag = "-Dmaven.repo.local=" <> mvnDir
 
+-- | Bazel tests are run in a bash environment with cmd.exe not in PATH. This results in ShellCommand
```

Comment:
Is that not fixable?

Reply:
Haven’t investigated it too deeply but so far I haven’t found a nice solution.